src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/AttendanceService.kt

```diff
@@ -30,10 +31,16 @@ class AttendanceService(
       val nomisCourseActivityId = mappingService.getMappingGivenActivityScheduleId(attendanceSync.activityScheduleId).nomisCourseActivityId
         .also { telemetryMap["nomisCourseActivityId"] = it.toString() }
 
+      val nomisAttendanceStatus = if (attendanceSync.status == "LOCKED") {
+        getNomisAttendanceStatus(nomisCourseActivityId, attendanceSync)?.eventStatus
+      } else {
+        null
+      }
```

Comment:
We only need the current Nomis attendance status if the Activities status is locked. Could have pushed this down into the method where `nomisAttendanceStatus` is used but that would involve passing various parameters down the call chain which just clutter things up - so chose to do it up front.